python/ray/tests/test_node_labels.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sys
+import pytest
+
+import ray
+
+
+@pytest.mark.parametrize(
+    "call_ray_start",
+    ['ray start --head --labels={"gpu_type":"A100","region":"us"}'],
```

Comment:
can you also test edge cases like the label key and value contain whitespace etc

Reply:
1. I add  a new 'test_ray_start_set_empty_node_labels' edge cases.
2. I will add more test cases for parameter exceptions in the next PR, like the case which the label key and value contain whitespace